clusterloader2/testing/network/config.yaml

```diff
@@ -2,7 +2,7 @@
 {{$NUMBER_OF_SERVERS := .CL2_NUMBER_OF_SERVERS}}
 {{$NUMBER_OF_CLIENTS := .CL2_NUMBER_OF_CLIENTS}}
 
-name: Network Performance
+name: network_performance 
```

Comment:
Why do you change this?

Reply:
2 reasons : 1) Other tests in cluster loader where there are 2 words they are not using capital letters(eg node-throughput). But just noticed they use small letters and words connected by "-" rather than "_".Shall I change this also to have "-" instead of "_"?
2)The file generated would have this as part of it's name , and having space just for this would seem awkward.

Reply:
ok - that's fine